### PR TITLE
Fix/deps update all

### PR DESCRIPTION
**Pull request by thusnjak**
* fixes warnings
* does deps update --all
* adds test config for rmq

Tests still don't work :(

### CODE REVIEW

**Review by hangyas — APPROVED** [review]

File: config/test.exs [hunk 1]
```diff
@@ -2,3 +2,11 @@ import Config
 
 config :logger, backends: [{LoggerFileBackend, :log_file}]
 config :logger, :log_file, level: :debug, path: "logs/test.log"
+
+config :task_bunny,
+  disable_auto_start: false,
+  hosts: [
+    default: [
+      connect_options: "amqp://rabbitmq:rabbitmq@localhost:5672/portal?heartbeat=30"
```
Comment (davorbadrov) [inline config/test.exs]: Should this connect to the portal or default host?

Comment (thusnjak) [reply config/test.exs]: Any :)

Comment (thusnjak) [inline config/test.exs]:
```suggestion
      connect_options: "amqp://rabbitmq:rabbitmq@localhost:5672?heartbeat=30"
```

Comment (davorbadrov) [reply config/test.exs]: This is my test output, it connects to either host, but my RMQ is configured to accept both guest and rabbitmq users:
RMQ_VHosts



Test output - still a lot of tests failing:
[task_bunny_test_output.txt](https://github.com/tsg-global/task-bunny/files/14297151/task_bunny_test_output.txt)

**Pull request merged**